src/main/java/com/shurjomukhi/Shurjopay.java

```diff
@@ -293,52 +217,43 @@ public PaymentReq getDefaultInfo(PaymentReq paymentReq) {
 			paymentReq.setClientIp(InetAddress.getLocalHost().getHostAddress());
 			
 		} catch (UnknownHostException e) {
-			
 			log.warn("Client IP does not found. Setting default ip address..", e);
-			paymentReq.setClientIp(DEFAULT_IP);
+			paymentReq.setClientIp("127.0.0.1");

```

Comment:
why fixed client ip ?? not acceptable

Reply:
Set client IP dynamically that's get from InetAddress. If face exception then handle it by hardcoded. Now clientIp default value set form payment req model class.